src/main/java/com/citusdata/migration/DynamoDBTableReplicator.java

```diff
@@ -452,6 +453,9 @@ void processRecords(List<Record> records) {
 //				LOG.debug(keyValue.toDelete());
 //				break;
 //			}
+			if (!Arrays.asList(TableSchema.EVENT_INSERT, TableSchema.EVENT_MODIFY, TableSchema.EVENT_REMOVE).contains(dynamoRecord.getEventName())) {
```

Comment:
The event name equals 'REMOVE' in this case 
![image](https://user-images.githubusercontent.com/1707627/50224821-d225f400-03d9-11e9-996b-42aa087cd06f.png)

Reply:
Propose
![image](https://user-images.githubusercontent.com/1707627/50225148-ab1bf200-03da-11e9-82d2-9816e0532354.png)

Reply:
Updated. But does this mean that we will not remove or mark record is removed when record is removed in dynamodb ?

Reply:
this issue happens when streaming has no data, it still marks the record as removed if having streaming data. 

not sure why the eventName equals to 'REMOVE' (should empty or null) when streaming has no data, may be it is a cache value.